docs/nextmv/tutorials/input.md

```diff
@@ -55,3 +67,124 @@ print(csv_archive_input_1.data)
 csv_archive_input_2 = nextmv.load(input_format=nextmv.InputFormat.CSV_ARCHIVE, path="custom_dir")
 print(csv_archive_input_2.data)
 ```
+
+## `MULTI_FILE` inputs
```

Comment:
I used the `DataFile` class with `input_data_key`, which felt necessary for a multi-sheet Excel, in my multi-excel example app [here](https://github.com/nextmv-io/community-apps/blob/merschformann/python-multi-excel/python-ortools-multiknapsack-multiexcel/main.py#L32-L43). Maybe we can add a custom example like that here as well? I actually liked working with it quite a bit. And again, I think it's necessary due to overlapping file/key for multi-sheet Excel (which are quite the norm).
Btw, I hope the example app mostly makes sense to you. It is mainly aligned with the Java one that came first, but maybe it needs another spin. :grimacing:

Reply:
Thank you for your suggestion, I added an example using the `input_data_key`.